package.json

```diff
@@ -1,34 +1,61 @@
 {
-  "name": "rainbow-argolia-search",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
-    "react-scripts": "3.3.1"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
+    "name": "rainbow-argolia-search",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@testing-library/jest-dom": "^4.2.4",
+        "@testing-library/react": "^9.3.2",
+        "@testing-library/user-event": "^7.1.2",
+        "react": "^16.12.0",
+        "react-dom": "^16.12.0",
+        "react-scripts": "3.3.1",
+        "styled-components": "^5.0.1"
+    },
+    "scripts": {
+        "start": "react-scripts start",
+        "build": "react-scripts build",
+        "test": "react-scripts test --watchAll=false --verbose",
+        "lint:js": "eslint src",
+        "lint:styles": "stylelint 'src/**/*.js'",
+        "lint": "yarn lint:js && yarn lint:styles",
+        "format": "prettier --write 'src/**/*.js'"
+    },
+    "devDependencies": {
+        "eslint-config-airbnb": "^18.0.1",
+        "eslint-config-prettier": "^6.10.0",
+        "eslint-config-react": "^1.1.7",
+        "eslint-plugin-jest": "^23.7.0",
+        "eslint-plugin-prettier": "^3.1.2",
+        "firebase-tools": "^7.13.0",
+        "husky": "^4.2.2",
+        "pre-push": "^0.1.1",
+        "prettier": "^1.19.1",
+        "prettier-eslint": "^9.0.1",
+        "pretty-quick": "^2.0.1",
+        "stylelint": "^13.1.0",
+        "stylelint-config-standard": "^20.0.0",
+        "stylelint-config-styled-components": "^0.1.1",
+        "stylelint-processor-styled-components": "^1.9.0"
+    },
+    "pre-push": [
+        "lint",
+        "test"
     ],
```

Comment:
since you have husky installed pre-push is not needed then you can:
```
   "husky": {
        "hooks": {
            "pre-commit": "pretty-quick --staged",
            "pre-push": "yarn lint && yarn test"
        }
    }
```